docs/api.md

```diff
@@ -32,11 +32,11 @@
 ::: httpx.delete
     :docstring:
 
-## `AsyncClient`
+## `Client`
 
-::: httpx.AsyncClient
+::: httpx.Client
     :docstring:
-    :members: headers cookies params request get head options post put patch delete build_request send aclose
+    :members: headers cookies params request get head options post put patch delete build_request send
```

Comment:
Don't we want to document `.close()` here?

Reply:
Thanks yup, now in.